megcruzen/browserify_productOrdering
# Added review styling and set up product.js to import from review.js

# Description
Added review styling and set up `product.js` to import from `review.js`. Review section currently displays on DOM (within product section) with one (hardcoded) review.

## Type of change

# Testing Instructions for Change Made
1. `git fetch --all`
2. `git checkout add-review-styling`
3. Run server/grunt
4. Verify that a review section (with one review) displays on DOM inside the product box.

# Checklist:
